python/lsst/cp/pipe/ptc/cpExtractPtcTask.py

```diff
@@ -343,22 +360,12 @@ def run(self, inputExp, inputDims):
                                                expIdMask=[expIdMask], covArray=covArray,
                                                covSqrtWeights=covSqrtWeights)
             # Use location of exp1 to save PTC dataset from (exp1, exp2) pair.
-            # expId1 and expId2, as returned by getInfo().getVisitInfo().getExposureId(),
-            # and the exposure IDs stured in inoutDims,
-            # may have the zero-padded detector number appended at
-            # the end (in gen3). A temporary fix is to consider expId//1000 and/or
-            # inputDims//1000.
-            # Below, np.where(expId1 == np.array(inputDims)) (and the other analogous
-            # comparisons) returns a tuple with a single-element array, so [0][0]
+            # Below, np.where(expId1 == np.array(inputDims))  returns a tuple
+            # with a single-element array, so [0][0]
```

Comment:
This logic would also be replaced by simply pulling the exposure values at the same time as the exposures in the loop above.